app/src/forms/minesattestations/dataService.js

```diff
@@ -119,16 +119,134 @@ const dataService = {
   readSubmission: async (confirmationId) => {
     return Models.Submission.query()
       .findOne({confirmationId: confirmationId})
-      .withGraphFetched({
-        attestation: true,
-        business: true,
-        contacts: true,
-        location: true,
-        statuses: true
-      })
+      .allowGraph('[attestation, business, contacts, location, statuses, statuses.notes, notes]')
+      .withGraphFetched('[attestation, business, contacts, location]')
+      .withGraphFetched('notes(orderDescending)')
+      .withGraphFetched('statuses(orderDescending).notes(orderDescending)')
       .throwIfNotFound();
-  }
+  },
+
+  updateSubmission: async (obj, user) => {
+    return {...obj, ...user};
+  },
+
+  allSubmissions: async () => {
+    const results = Models.Submission.query()
+      .allowGraph('[attestation, business, contacts, location, statuses, statuses.notes, notes]')
+      .withGraphFetched('[attestation, business, contacts, location]')
+      .withGraphFetched('notes(orderDescending)')
+      .withGraphFetched('statuses(orderDescending).notes(orderDescending)');
+    return results || [];
+  },
+
+  createSubmissionStatus: async (obj, confirmationId, user) => {
+    if (!obj) {
+      throw Error('Status cannot be created without data');
+    }
+    let trx;
+    let result;
```

Comment:
Result can be declared as a const inside the try block as it's only used there on return. You could actually even get away with just return awaiting line 160 with line 162.